Formula/safe-rm.rb

```diff
@@ -1,8 +1,9 @@
 class SafeRm < Formula
   desc "Wraps rm to prevent dangerous deletion of files"
   homepage "https://launchpad.net/safe-rm"
-  url "https://launchpad.net/safe-rm/trunk/0.12/+download/safe-rm-0.12.tar.gz"
-  sha256 "1c9d3113591e249301fd00fff51152069ab71cd518b32bfcf6848a8d6c3054e2"
+  url "https://launchpad.net/safe-rm/trunk/0.13/+download/safe-rm-0.13.tar.gz"
+  sha256 "f0f792faa076fbf3dbffbb4bc8918fd6bf699775417fef3f5f408f0c69200c12"
+  license "GPL-3.0-only"
```

Comment:
```suggestion
  license "GPL-3.0-or-later"
```

According to `README.md`:

```
Copyright (C) 2008-2020  Francois Marier <francois@fmarier.org>

This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program.  If not, see <https://www.gnu.org/licenses/>.
```